gwadoc/doc_en.py

```diff
@@ -924,7 +925,7 @@
 ### Relation: holo_part EDP27
 
 relations.holo_part.name.en = "Part Holonym"
-relations.holo_part.df.en = "Concept B is the whole of parts of Concept A"
+relations.holo_part.df.en = "Concept B is the whole parts of Concept A"
```

Comment:
I don't think this is better, because now it sounds like B is a part of A, but it should say that B is the whole where A is a part.

Reply:
Thanks. Will correct that.